.github/workflows/update-deps.yaml

```
@@ -60,5 +81,6 @@ jobs:
           signoff: true
           delete-branch: true
           base: main
-          title: "Update Actions dependency"
-          body: "Automated pull request to update Actions."
+          title: "Update libs dependency"
+          body: "Automated pull request to update Dependencies."
+          labels: ${{ steps.pr-label.LABELS }}
```

actionlint

property "labels" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)